Added recommendations and fixed typos (#278)
Azure/Azure-Proactive-Resiliency-Library: docs/content/services/specialized-workloads/azure-virtual-desktop/_index.md

| [AVD-5 Implement RDP Shortpath for Public or Managed Networks](#avd-5---implement-rdp-shortpath-for-public-or-managed-networks)                                             |    Networking     |  Medium  | Preview |         No          |
| [AVD-6 Implement a Multi-Region BCDR Plan](#avd-6---implement-a-multi-region-bcdr-plan)                                                                                     | Disaster Recovery |  Medium  | Preview |         No          |
| [AVD-7 Store Golden Image Redundantly for Disaster Recovery](#avd-7---store-golden-image-redundantly-for-disaster-recovery)                                                 | Disaster Recovery |   Low    | Preview |         No          |
| [AVD-8 Capacity Planning for AVD Resources](#avd-8---capacity-planning-for-avd-resources)                                                                                   | Disaster Recovery |   Low    | Preview |         No          |
| [AVD-9 Ensure that FSLogix Storage Account is Redundant](#avd-9---ensure-that-fslogix-storage-account-is-redundant)                                                         |   Availability    |   High   | Preview |         No          |

> | [AVD-10 Enable Azure Backup for FSLogix Storage Account](#avd-10---enable-azure-backup-for-fslogix-storage-account)                                                         |   Backup/Storage    |   Medium   | Preview |         No          |

Reviewer: ```suggestion
| [AVD-10 Enable Azure Backup for FSLogix Storage Account](#avd-10---enable-azure-backup-for-fslogix-storage-account)                                                         |   Disaster Recovery    |   Medium   | Preview |         No          |
```

Reviewer: Switching this to align to https://azure.github.io/Azure-Proactive-Resiliency-Library/contributing/#recommendation-categories

<br><br>

### AVD-4 - Deploy Domain Controllers and DNS Servers in Azure Virtual Network Across Availability Zones

**Category: Availability**

**Impact: Medium**

**Guidance**

> When using an AD DS identity solution with AVD, it is recommended to deploy domain controllers and DNS servers on Azure virtual machines across availability zones. This improves the environment’s reliability by being independent of an on-premises and creating a shorter path for user authentication improving performance.

Reviewer 2: ```suggestion
When using an AD DS identity solution with AVD, it is recommended to deploy domain controllers and DNS servers on Azure virtual machines across availability zones. This improves the environment’s reliability by removing a dependency on an on-premises service and improves performance by creating a shorter path for user authentication.
```
